stable/spark/values.yaml

```diff
@@ -43,16 +49,19 @@ Worker:
   ReplicasMax: 10
   CpuTargetPercentage: 50
 
+
 Zeppelin:
   Name: zeppelin
-  Image: "apache/zeppelin"
-  ImageTag: "0.7.3"
+  Image: "dbanda/zeppelin"
```

Comment:
Container image source

Reply:
https://github.com/Azure/mmlspark/tree/master/tools/helm/docker_images
We also have them referenced in the chart README.md 
https://github.com/helm/charts/pull/11285/files#diff-bfd7557b276fc06ebac03d615513291cR144